backend/pkg/logger/logger.go

```diff
@@ -0,0 +1,79 @@
+package logger
+
+import (
+	"fmt"
+	"sync"
+	"sync/atomic"
+
+	"github.com/HyperloopUPV-H8/h9-backend/pkg/abstraction"
+)
+
+// Logger is a struct that implements the abstraction.Logger interface
+type Logger struct {
+	// An atomic boolean is used in order to use CompareAndSwap in the Start and Stop methods
+	running        *atomic.Bool
+	subloggersLock sync.RWMutex
+	subloggers     map[abstraction.LoggerName]abstraction.Logger
+}
+
+func (logger *Logger) Start(startKeys []abstraction.LoggerName) {
```

Comment:
this returns `error` as you defined in the `abstraction.Logger`
I recommend you add something like
```go
var _ abstraction.Logger = &Logger{}
```
to ensure at every moment that your code follows the interface